Signal/src/Models/OWSMessagesBubblesSizeCalculator.m

```diff
@@ -56,6 +56,10 @@ - (instancetype)init
         _referenceSystemMessageCell = [OWSSystemMessageCell new];
         _referenceUnreadIndicatorCell = [OWSUnreadIndicatorCell new];
         _referenceContactOffersCell = [OWSContactOffersCell new];
+
+        // Calculating message size is relatively expensive, so unbound the size of the cache.
+        self.cache.countLimit = 0;
+        self.cache.totalCostLimit = 0;
```

Comment:
I don't foresee this "hash-to-NSValues containing CGSizes" cache becoming that large in memory and its a huge perf hotspot.